qutebrowser/completion/models/miscmodels.py

```diff
@@ -21,7 +21,10 @@
 
 from qutebrowser.config import configdata
 from qutebrowser.utils import objreg, log
+import itertools
 from qutebrowser.completion.models import completionmodel, listcategory, util
+from qutebrowser.misc import tabsfinder
+from qutebrowser.misc.tabsfinder import delete_tab
```

Comment:
Please only import modules, not things form inside modules. With the import above, you can just use `tabsfinder.delete_tab` anyways.